src/js/controllers/preferences.js

```diff
@@ -89,9 +89,6 @@ angular.module('copayApp.controllers').controller('preferencesController',
         value: $scope.wallet.balanceHidden
       };
 
-      if (wallet.isPrivKeyExternal)
-        $scope.externalSource = wallet.getPrivKeyExternalSourceName() == 'ledger' ? 'Ledger' : 'Trezor';
```

Comment:
This option is not used in the view, however we must hide all security options from waller preferences if it is an external source.